packages/list_bloc/lib/src/cubits/list_cubit.dart

```diff
@@ -20,5 +20,50 @@ class ListCubit<T, F> extends DataCubit<List<T>, F> {
       rethrow;
     }
   }
-// TODO(alexis): implement remove, add, replace etc
+
+  void add(T element) {
+    final data = <T>[...state.data ?? [], element];
+    emit(state.copyWith(data: data));
+  }
+
+  void remove(T element) {
+    final data = <T>[...state.data ?? []]..remove(element);
```

Comment:
Here if the `remove` returned `false` meaning no item was removed from the list, we should also throw an error regarding that removal failure.